etc/org.opencastproject.ingest.impl.IngestServiceImpl.cfg

```diff
@@ -54,3 +54,9 @@
 #org.opencastproject.download.source = http://localhost/.*
 #org.opencastproject.download.user = opencast_system_account
 #org.opencastproject.download.password = CHANGE_ME
+
+# Control if events without series are added to an auto generated CA series. This ensures that an event without series is accessible for users with rights to the CA.
+#
+# Default: false
```

Comment:
I think this does something else. It's a prefix, not a boolean. Though, having this unset will disable the functionality. Also, the default is not `false` but EMPTY.